src/utils/error.ts

```diff
@@ -1,3 +1,47 @@
+import { SharedModels } from "@kontent-ai/management-sdk";
+import { isAxiosError } from "axios";
+
 export const throwError = (msg: string) => {
   throw new Error(msg);
 };
+
+export type KontentError = Readonly<{
+  kontentErrorCode: number;
+  message: string;
+  validationMessages: ReadonlyArray<string>;
+}>;
+
+export const skipKontentErrors = (errorCodesToSkip: ReadonlyArray<number>) =>
```

Comment:
same with this funciton, it is not used anywhere, but might be possibly useful in future :D was this the intention?

Reply:
Yeah, I think it may be useful in the future. For now, I think we can use it to ignore the 404 error when we are checking for existence of items and assets in validation.